include/enums/cpp_memory_layout.h

```diff
@@ -51,19 +52,15 @@ inline MemoryLayout convert_layout(CMemoryLayout layout) {
   direct use of MemoryLayout.
   */
 
-  switch(layout) {
-    case(c_nested) :
+  switch (layout) {
```

Comment:
We should probably add ``invalid`` to the enum conversion switch cases.

Reply:
I'm surprised I didn't get a warning (and thus fail to compile) with this